src/plugins/walletConnectToDapps/components/modal/callRequest/useCallRequestFees.ts

```diff
@@ -0,0 +1,74 @@
+import type { EvmBaseAdapter, EvmChainId } from '@shapeshiftoss/chain-adapters'
+import { FeeDataKey } from '@shapeshiftoss/chain-adapters'
+import type { WalletConnectEthSendTransactionCallRequest } from '@shapeshiftoss/hdwallet-walletconnect-bridge/dist/types'
+import { useWalletConnect } from 'plugins/walletConnectToDapps/WalletConnectBridgeContext'
+import { useCallback, useEffect, useState } from 'react'
+import type { FeePrice } from 'components/Modals/Send/views/Confirm'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+
+export function useCallRequestFees(
+  request: WalletConnectEthSendTransactionCallRequest['params'][number],
+) {
+  const [fees, setFees] = useState<FeePrice | undefined>({
+    [FeeDataKey.Slow]: { txFee: '0.0001', fiatFee: '5.00' },
```

Comment:
we should never default to low fees like this, if these ever get consumed we're going to get stuck txs